src/instruments/yokogawa/yokogawa6370.py

```diff
@@ -100,20 +114,49 @@ class Traces(Enum):
 
     # PROPERTIES #
 
+    # General
+
+    id = string_property(
+        "*IDN",
+        doc="""
+            Get the identification of the device.
+            Output: 'Manufacturer,Product,SerialNumber,FirmwareVersion'
+            Sample: 'YOKOGAWA,AQ6370D,90Y403996,02.08'
+            """,
+        readonly=True,
+    )
+
+    status = unitless_property(
+        "*STB", doc="""The status byte of the device.""", readonly=True
```

Comment:
Maybe specify description of what the status byte is.

Reply:
I added a description, however the status is not very useful.